### PR TITLE
Feature/cov 241

### PR DESCRIPTION
**Pull request by deltaDirk**
**Type**
Feature
**Description**
URL Shortening and adjustment of SMS flow.
**Related tickets**
COV-241, COV-323

### CODE REVIEW

File: V1/Brokers/BitlyBroker.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        }
+
+        public async Task<string> ShortenRequest(object payload)
+        {
+            var response = await _httpClient.PostAsJsonAsync(_partialRoot, payload);
+
+            if (!response.IsSuccessStatusCode)
+                await ProcessFailureMessage(response);
+
+            return await response.Content.ReadAsStringAsync();
```
Comment (Roleen20) [inline V1/Brokers/BitlyBroker.cs:30]: Desterilizer the string here and sent the Bitly response back. Better to do it in the broker

File: V1/Constants/Messages.cs [hunk 2]
```diff
@@ -50,7 +50,9 @@ public class Messages
         public static readonly string Val_MobileNumber = $"Invalid mobile number.";
         public static readonly string Val_Identification = $"Invalid Identification number.";
         public static readonly string Val_DateNotInPast = $"Invalid Date, date must be in the past";
-        public static readonly string Val_FileTooLarge = $"The File size is to large.";
+        public static readonly string Val_FileTooLarge = $"The file size is to large.";
```
Comment (Roleen20) [inline V1/Constants/Messages.cs:53]: "too"

File: V1/Interfaces/Repositories/IWalletRepository.cs [hunk 3]
```diff
@@ -6,5 +6,6 @@ namespace CoviIDApiCore.V1.Interfaces.Repositories
 {
     public interface IWalletRepository: IBaseRepository<Wallet, Guid>
     {
+        Task<Wallet> GetWalletByMobileNumberAndIndicator(string mobileNumber);
```
Comment (Roleen20) [inline V1/Interfaces/Repositories/IWalletRepository.cs:9]: Remove "AndIndicator"

Comment (deltaDirk) [reply V1/Interfaces/Repositories/IWalletRepository.cs:9]: We're not doing the indicator anymore

File: V1/Services/WalletService.cs [hunk 4]
```diff
@@ -68,6 +68,11 @@ public async Task<TokenResponse> CreateWallet(CreateWalletRequest walletRequest)
         {
             var otpReturn = await _otpService.GenerateAndSendOtpAsync(walletRequest.MobileNumber);
 
+            var mobileWallet = await _walletRepository.GetWalletByMobileNumberAndIndicator(walletRequest.MobileNumber);
+
```
[the remark this answers is not in the document]
Comment (deltaDirk) [reply V1/Services/WalletService.cs:72]: This was then done after this PR was opened and not reviewed.

**Pull request merged**